### DISCUSSION TITLE
Lock Event - Additional Details in Home Assistant

### DISCUSSION THREAD

**Discussion by jgillardgit** [jason0x43/hacs-hubitat#206] [Q&A]
First of all, I'd like to thank all of the contributors for the great work that you do.

I'm wondering if it would be possible to send additional event details to Home Assistant when a lock is unlocked or locked.

In the Hubitat event log, I see:

> Porch Door Lock was unlocked by <Name>
or
> Porch Door Lock was manually locked [physical]

Would be great if I could see these details in HA so that I can trigger other automations based on this.

**Comment by MatthewGreen** [comment]
I'm pretty curious about this as well. The documentation mentions event emitting devices [here](https://github.com/jason0x43/hacs-hubitat#event-emitting-devices). Specifically this has me curious: _In Node Red, a workflow can listen for hubitat_event events and filter them based on properties in payload.event._

When I went to Developer Tools and "Started Listening" for *hubitat_event* I didn't see anything happen after locking/unlocking the Z-Wave Hubitat Lock.

@jason0x43 what are we missing here?
